example/android/app/src/main/AndroidManifest.xml

```diff
@@ -23,7 +23,6 @@
     <!-- Android Auto specific permissions -->
     <uses-permission android:name="androidx.car.app.NAVIGATION_TEMPLATES" />
     <uses-permission android:name="androidx.car.app.ACCESS_SURFACE" />
-    <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
```

Comment:
This just removes duplicate row from the file.
same permission is given few lines before.